README.md

```diff
@@ -101,13 +101,15 @@ PostCSS Plugins.
 
 ### inject
 
-Type: `boolean` `object`<br>
+Type: `boolean` `object` `function(file): string`<br>
```

Comment:
what is argument `file` used for?

Reply:
The given function will receive the file params, allowing the user to compile or inject their own css using the same name as the input etc.

Reply:
For example HMR support :)